cmd/argocd-server/commands/root.go

```diff
@@ -146,6 +148,7 @@ func NewCommand() *cobra.Command {
 	command.Flags().StringVar(&repoServerAddress, "repo-server", common.DefaultRepoServerAddr, "Repo server address")
 	command.Flags().StringVar(&dexServerAddress, "dex-server", common.DefaultDexServerAddr, "Dex server address")
 	command.Flags().BoolVar(&disableAuth, "disable-auth", false, "Disable client authentication")
+	command.Flags().BoolVar(&disableGZip, "disable-gzip", false, "Disable GZIP compression")
```

Comment:
Given that the current behaviour in master doesn't include gzip compression, we should keep the default as `true` for sometime, before considering turning it on as a default for everyone?